### ISSUE TITLE
Create release and publish to crates.io

### ISSUE THREAD

**Issue by j-richey** [j-richey/open_ttt_lib#6]
Create a release and publish the library to crates.io so others have access to it. This is described in the [Ounce of Rust Project Manual](https://j-richey.github.io/project-documentation/ounce-of-rust/) section [4.3.2. Publishing to crates.io](https://j-richey.github.io/project-documentation/ounce-of-rust/design/distribution.html#publishing-to-crates-io).

* Create crates.io account
* Ensure all meta data is set correctly in `Cargo.toml`
* Create a release entry in `CHANGELOG.md`.
* Update the `README.md` to point to the crate location
* Ensure there is a getting started example in the documentation 
* Create a release tag
* Verify docs.rs picked up the documentation

**Comment by j-richey** [comment]
The Rust [Documentation guidelines](https://rust-lang.github.io/api-guidelines/documentation.html) mentioned using the [version-sync](https://crates.io/crates/version-sync) to ensure the `README.md` and `html_root_url` attribute stay synchronized with the `Cargo.toml` file.

**Comment by j-richey** [comment]
It looks like the `Cargo.toml` file allows badges to be specified. Here is an example from [version-sync](https://github.com/mgeisler/version-sync/blob/master/Cargo.toml): 

```toml
# Other sections here...
[badges]
travis-ci = { repository = "mgeisler/version-sync" }
appveyor = { repository = "mgeisler/version-sync" }
codecov = { repository = "mgeisler/version-sync" }
```

**Comment by j-richey** [comment]
It is really nice when there is a prominent example of how to use a library in the `README.md`. However, this can become out of sync with the library or contain codes. The [doc-comment](https://crates.io/crates/doc-comment) crate can be used to test code examples in the README file.

**Comment by j-richey** [comment]
The changes merged in pull request #24 did not include updating the `README.md` file on how to run the benchmarks. This should be done as part of this task.

**Comment by j-richey** [comment]
The crate has been published to crates.io! The first version published forgot to set the `readme` key in the `Cargo.toml` file. That was fixed in version v0.1.1.
